scripts/dynomite

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
I do not think we need this script as we already have it in https://github.com/Netflix/dynomite/blob/dev/bin/launch_dynomite.sh 
Are we trying to do something different?